pkg/karmadactl/addons/metricsadapter/metricsadapter.go

```diff
@@ -169,6 +170,13 @@ func installComponentsOnKarmadaControlPlane(opts *addoninit.CommandAddonsEnableO
 		return fmt.Errorf("error when parsing karmada metrics adapter AA service template :%v", err)
 	}
 
+	const karmadaCertsName = "karmada-cert"
+	const caCertAndKeyName = "ca.crt"
```

Comment:
Can we use the common definition of the certs name and key name?
Although we have no plans to update the cert name used by `karmada-apiserver`, but referencing the same const would make the code more readable.

Reply:
I updated the code to use the global const variable within `karmadactl` module.

Actually I want to do more modification:

1. I think the secret name and its keys of karmada certs should be a whole group, 
    but now `KarmadaCertsName` and `CaCertAndKeyName`lies in `pkg/karmadactl/options/global.go`, 
    while others lies in `pkg/karmadactl/cmdinit/options/global.go`, 
    actually I want to put them all to `pkg/karmadactl/options/global.go`.

2. Now ca cert name and key name is defined in just a variable `CaCertAndKeyName`,
    when getting ca certName using `fmt.Sprintf("%s.crt", CaCertAndKeyName)`,
    when getting ca keyName using `fmt.Sprintf("%s.key", CaCertAndKeyName)`,
    to some extend, the file suffix is hard code, I think we'd better split the valiable into two,
    just `CaCertName=ca.crt` and `CaKeyName=ca.key`

do you think it is necessary to do the above modification?